### ISSUE TITLE
load_env.sh is hanging on new clone of repo

### ISSUE THREAD

**Issue by behrica** [microsoft/sample-app-aoai-chatGPT#59]
As as this point no azd environment is existing, the `azd env get-values` wait for user input, which makes teh script wait for ever

https://github.com/microsoft/sample-app-aoai-chatGPT/blob/ad91c29e2b50e40bf2f1d50bc9f59bba23a889cb/scripts/loadenv.sh#L3

**Comment by behrica** [comment]
I don't do "azd up" ...

To run a local copy of the code, this is not really required, is it ?
Running "start.sh" would work, if it would not try to run "loadenv.sh" which hangs.

I did set the needed environment variables  in bash before doing :/start


So this sequence will fail:
1. git clone
2. export xxxx (all need env vars)
3.  . start.sh

In my case I did setup already all need dependencies before (openai and Cognitive Search)
